salt/_states/docker_registry.py

```diff
@@ -0,0 +1,202 @@
+'''
+States to manage images in a Docker registry.
+'''
+
+import logging
+
+import salt.exceptions
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'docker_registry'
+
+
+def __virtual__():
+    if 'docker_registry.get_image_manifest' not in __salt__:
+        return (False, "Missing 'docker_registry' module")
+
+    return __virtualname__
+
+
+def image_managed(name, archive_path=None, digest=None, force=False,
```

Comment:
minor (layout): isn't

```
def image_managed(name, archive_path=None, digest=None, force=False,
                  tls_verify=True):
```

or

```
def image_managed(
    name, archive_path=None, digest=None, force=False,tls_verify=True
):
```

more legible? Or maybe it's a salt coding style again (if so, feel free to ignore this comment).

Reply:
Applying what I remember from PEP8 in the OpenStack days. As long as we don't have automated linting, anything that doesn't look too crazy is good to go. Otherwise, de gustibus and bikeshedding.

Made a modification to the code (will be in another PR, this one is abandoned, hang on).